src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,46 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final String REPORT_FOR_PERIOD = "Report for period";
+    private static final String WHITESPACE = " ";
+    private static final String DASH = "-";
+    private static final String PATTERN = "d.MM.yyyy";
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern(PATTERN);
+    private static final int INDEX_OF_DATE = 0;
+    private static final int INDEX_OF_WORKING_HOUR = 2;
+    private static final int INDEX_OF_SALARY_PER_HOUR = 3;
+    private static final int DAY_INCLUSIVE = 1;
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
```

Comment:
Divide program logic
Large methods is very difficult to read and understand. So methods should be short, between 5-15 lines. And it’s important for you to remember that one method should be responsible for one task.